### ISSUE TITLE
Help questions.

### ISSUE THREAD

**Issue by Nopalbackdoor99** [meganz/webclient#123]
Hello, can you help me by answering some questions about privacy. I have already tried mega before I had a pro Lite account. But now I want to use it for my main cloud. I will store my music, (legal)porn, movies and renders. 
1.My data is safe here and I mean safe from being deleted?

 All my content is legal and they are not even piracy, I have paid on the internet for them so it would hurt me to lose them.

 I just want to ask users who have been using mega for years, If I upload legal and copyrighted content to my private cloud I will not share it with anyone, can't it be a deletion salvation? I understand that people who complain about that is why they have shared or downloaded content like this, but I am not going to share or anything. 

In other words, I don't care if mega sees my porn, on the contrary that they enjoy it, but I would not like it to be eliminated since it has cost me and I am not sharing it with anyone. I

'm only going to use it as a backup for my laptop since I can't save 7TB there 

I hope you can answer me please. 

I know that no company is exempt from going bankrupt, but eligi mega doesn't look like it's going to go bankrupt tomorrow. 

Thanks in advance 

M.

**Comment by rn-mega** [comment]
MEGA Copyright policy:
MEGA respects the copyrights of others and requires that users of the MEGA cloud service comply with the laws of copyright. You are strictly prohibited from using the MEGA cloud service to infringe copyrights. You may not upload, download, store, share, display, stream, distribute, e-mail, link to, communicate, transmit or otherwise make available any files, data, or content that infringes any copyright or other proprietary rights of any person or entity. For the full policy wording click here: https://mega.nz/copyright

**Comment by gabianastasie** [comment]
@Nopalbackdoor99 Boycott MEGA! [https://web.archive.org/web/20231101003041/https://github.com/meganz/webclient/issues/135](https://web.archive.org/web/20231101003041/https://github.com/meganz/webclient/issues/135)

MEGA desperately tries to cover up abuse. MEGA even deleted the above-mentioned post, but we used archive.org to save it.

Don't give your money to low-down individuals that harm a mentally ill person. Don't associate yourself with such scoundrels.
